extension/src/core/config.js

```diff
@@ -18,75 +18,75 @@ const makeCSS = ({ name, color }) => `
 `
 
 const defaultGetter = {
-    draft_color: DEFAULT_DRAFT_COLOR,
-    approved_color: DEFAULT_APPROVED_COLOR,
-    obsolete_color: DEFAULT_OBSOLETE_COLOR,
-    request_changes_color: DEFAULT_REQUEST_CHANGES_COLOR,
-    review_required_color: DEFAULT_REVIEW_REQUIRED_COLOR,
-    default_color: 'transparent',
-    pr_obsolescence_in_day: DEFAULT_OBSOLESCENCE_IN_DAY,
+  draft_color: DEFAULT_DRAFT_COLOR,
+  approved_color: DEFAULT_APPROVED_COLOR,
+  obsolete_color: DEFAULT_OBSOLETE_COLOR,
+  request_changes_color: DEFAULT_REQUEST_CHANGES_COLOR,
+  review_required_color: DEFAULT_REVIEW_REQUIRED_COLOR,
+  default_color: 'transparent',
+  pr_obsolescence_in_day: DEFAULT_OBSOLESCENCE_IN_DAY,
```

Comment:
ta modif des l.52-59 ne colle pas avec ça :) 
```suggestion
  draftColor: DEFAULT_DRAFT_COLOR,
  approvedColor: DEFAULT_APPROVED_COLOR,
  obsoleteColor: DEFAULT_OBSOLETE_COLOR,
  requestChangesColor: DEFAULT_REQUEST_CHANGES_COLOR,
  reviewRequiredColor: DEFAULT_REVIEW_REQUIRED_COLOR,
  defaultColor: 'transparent',
  prObsolescenceInDay: DEFAULT_OBSOLESCENCE_IN_DAY,
```